client.go

```diff
@@ -449,3 +450,22 @@ func (c *Client) WriteStream(path string, stream io.Reader, _ os.FileMode) (err
 		return newPathError("WriteStream", path, s)
 	}
 }
+
+func (c *Client) WriteWrCls(_ context.Context, path string, _ os.FileMode) (writer io.WriteCloser, err error) {
```

Comment:
1. the naming could be improved
2. why is the context required if not used?
3. is it necessary to declare the err?, it`s always `nil`